README.md

```diff
@@ -1,112 +1,207 @@
 # Boulder Release Process Docs & Demo
 
 A repo for documenting and demonstrating the
-[Boulder](https://github.com/letsencrypt/boulder) release process used to move
-code from Github to [staging](https://letsencrypt.org/docs/staging-environment/)
-and [production](https://acme-v01.api.letsencrypt.org/).
+[Boulder](https://github.com/letsencrypt/boulder) release process used
```

Comment:
Looks like the wrapping wrapped to significantly less than 80 characters (was it accidentally set to 72?).

More generally, re-wrapping files like this is a great cleanup to do but it's best for it to be in a separate commit from content changes, so that it's easier to tell when substantive things changed vs just style things.

Reply:
Ah, yeah, my bad on both counts, I'll fix it.